plan/constants.py

```diff
@@ -84,6 +97,7 @@ class PlanData:
             "Unlimited private repositories",
             "Priority Support",
         ],
+        tier_name=TierName.PRO.value,
```

Comment:
I think the types are slightly off here.  `TierName.PRO` has type `TierName` but `TierName.PRO.value` has type `str`